COMMON_ISSUES.md

```diff
@@ -44,6 +44,14 @@ to ensure playback:
   video.muted = true;
   ```
 
+Chrome 76+ Group Room Participants downgrade outgoing video bitrate for high `maxAudioBitrate` values
```

Comment:
Should it be `downgraded` instead of `downgrade`?

Reply:
@syerrapragada We usually use the present tense for COMMON_ISSUES.md entries. We typically use past tense for bug fixes.